tests/async/test_browsertype_connect.py

```diff
@@ -0,0 +1,179 @@
+# Copyright (c) Microsoft Corporation.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import asyncio
+import os
+import signal
+import subprocess
+
+import pytest
+
+from playwright._impl._driver import compute_driver_executable
+from playwright.async_api import BrowserType, Error
+from tests.server import Server
+
+
+class RemoteServer:
+    def __init__(self, browser_name: str) -> None:
+        self.process = subprocess.Popen(
```

Comment:
Maybe use `asyncio` `create_subrocess_exec` as it is used on async context @mxschmitt

Reply:
Please do not mention me all the time with the @, this will result in a notification on my phone every time.
In this case I keep it because I want to use it in sync tests too. But thanks for your suggestions.

Reply:
Ok I won't @mention you onwards, It is better to keep current as you also wanna use it to test in `sync` context